torch/_dynamo/output_graph.py

```diff
@@ -2115,7 +2136,12 @@ def get_trace_call_log_str():
             ]
         elif kind == "call_module":
             if self.parent is not None:
-                unimplemented("Invoking an nn.Module inside HigherOrderOperator")
+                unimplemented_v2(
+                    gb_type="Invoking an nn.Module inside a HigherOrderOperator",
```

Comment:
I'm confused because this should be supported

Reply:
Also for all of the HOP ones, we should use the HOP name directly (e.g. autograd.Function) instead of calling it a HigherOrderOperator, users don't know what this is

Reply:
Offline discussion w/ @ydwu4: This should be supported - we can remove this graph break then (or at the very least mark that this graph break is not expected/likely a bug).

Reply:
This seems to be creating a call_module node inside the hop subgraph. Probably it's here before the in_nn_module flag because we don't want call_module to be in subgraph.  Can probably delete now.

Reply:
inline_inbuilt_nn_modules could still be unset - so this graph break could still occur then?

Reply:
I don't think there's a good reason for the user to set inline_inbuilt_nn_modules=False